bin/pixelmatch

```diff
@@ -8,14 +8,15 @@ const fs = require('fs');
 const match = require('../.');
 
 if (process.argv.length < 4) {
```

Comment:
@mourner any opposition to using something like yargs or meow to make the CLI functionality a bit more robust + support all options?

Reply:
I'm up for it, but let's leave it for another PR so that we don't conflate changes. Maybe it would be better to leave out `differentiateDiffs` from the CLI option for now since it's minimal and doesn't accept diff color options anyway.